.github/workflows/bindings.yml

```diff
@@ -175,31 +175,38 @@ jobs:
         id: setup-chrome
         with:
           chrome-version: stable
-      - uses: actions-rust-lang/setup-rust-toolchain@v1  # this implicitly caches Rust tools and build artifacts
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
         with:
-          rustflags: ''
           target: wasm32-unknown-unknown
       - uses: oven-sh/setup-bun@v2
         with:
           bun-version: latest
       - uses: actions/setup-node@v4
         with:
           node-version: 18
-      - name: Setup cargo-make
-        uses: davidB/rust-cargo-make@v1
       - name: Install wasm-pack
         uses: taiki-e/install-action@v2
         with:
           tool: wasm-pack
-      - name: Build & test WASM / JS package
+      - uses: davidB/rust-cargo-make@v1
+      - name: build WASM
         run: |
           cd crypto-ffi
           cargo make wasm
-          cd bindings/js
-          bun eslint . --max-warnings=0
-
-          # check all Typescript files (wdio.conf.ts, benches)
-          bunx tsc --noEmit
-
+      - name: Setup bun
```

Comment:
This should be named something like "Compile TS files", because bun is already set up by the time we are here. Here we are just installing dependencies and running the compiler + bundler.

Reply:
Sure, will adjust.